cpp/tests/community/induced_subgraph_test.cpp

```diff
@@ -178,6 +179,8 @@ class Tests_InducedSubgraph : public ::testing::TestWithParam<InducedSubgraph_Us
                         h_subgraph_vertices.data(),
                         h_subgraph_vertices.size(),
                         handle.get_stream());
+    raft::device_span<size_t const> dspan_subgraph_offsets(d_subgraph_offsets.data(), d_subgraph_offsets.size() );
+    raft::device_span<vertex_t const> dspan_subgraph_vertices(d_subgraph_vertices.data(), d_subgraph_vertices.size() );
```

Comment:
`d_` is a convention to distinguish device memory variable from host memory variable (starting with `h_`) if we're using otherwise the same variable name.

Including `span` in the variable name might be too redundant as the type of the variable clearly tells this.

Better just be "d_subgraph.." than "dspan_subgraph".

Reply:
And this applies to the code below as well.

Reply:
directly using d_subgraph.. may cause naming conflict, so I changed it to d_Subgraph..